src/odemis/driver/technolution.py

```diff
@@ -1145,23 +1153,29 @@ def _acquire(self):
                     dataContent = args[1]  # Specifies the type of image to return (empty, thumbnail or full)
                     notifier_func = args[2]  # Return function (usually, dataflow.notify or acquire_single_field queue)
 
-                    self.parent.asmApiPostCall("/scan/scan_field", 204, field_data.to_dict())
-
-                    if DATA_CONTENT_TO_ASM[dataContent] is None:
-                        da = model.DataArray(numpy.array([[0]], dtype=numpy.uint8), metadata=self._metadata)
-                    else:
-                        # TODO remove time.sleep if the function "waitOnFieldImage" exists. Otherwise the image is
-                        #  not yet loaded on the ASM when trying to retrieve it.
-                        time.sleep(0.5)
-                        resp = self.parent.asmApiGetCall(
-                                "/scan/field?x=%d&y=%d&thumbnail=%s" %
-                                (field_data.position_x, field_data.position_y,
-                                 str(DATA_CONTENT_TO_ASM[dataContent]).lower()),
-                                200, raw_response=True, stream=True)
-                        resp.raw.decode_content = True  # handle spurious Content-Encoding
-                        img = Image.open(BytesIO(base64.b64decode(resp.raw.data)))
-
-                        da = model.DataArray(img, metadata=self._metadata)
+                    try:
+                        self.parent.asmApiPostCall("/scan/scan_field", 204, field_data.to_dict())
+
+                        if DATA_CONTENT_TO_ASM[dataContent] is None:
+                            da = model.DataArray(numpy.array([[0]], dtype=numpy.uint8), metadata=self._metadata)
+                        else:
+                            # TODO remove time.sleep if the function "waitOnFieldImage" exists. Otherwise the image is
+                            #  not yet loaded on the ASM when trying to retrieve it.
+                            time.sleep(0.5)
+                            resp = self.parent.asmApiGetCall(
+                                    "/scan/field?x=%d&y=%d&thumbnail=%s" %
+                                    (field_data.position_x, field_data.position_y,
+                                     str(DATA_CONTENT_TO_ASM[dataContent]).lower()),
+                                    200, raw_response=True, stream=True)
+                            resp.raw.decode_content = True  # handle spurious Content-Encoding
+                            img = Image.open(BytesIO(base64.b64decode(resp.raw.data)))
+
+                            da = model.DataArray(img, metadata=self._metadata)
+                    except Exception as ex:
```

Comment:
The same as above:
>The only expected exception here would be the AsmApiException isn't it from asmApiPostCall? Should we not limit the catch to just that exception? Instead of catching all the exceptions? Even the error is already logged in that method.